Trackovid19-web/src/app/screens/main/main.component.scss

```diff
@@ -86,6 +86,27 @@
     display: none;
   }
 }
+.share-status {
+  display: block;
+  box-sizing: border-box;
+  font-weight: bold;
+  border-radius: 2.5rem;
+  outline: none;
+  cursor: pointer;
+  padding: 1.125rem 2.5rem;
+  margin: 0 12px;
+  border: 1px solid #fa6400;
+  font-family: Archivo;
```

Comment:
I think @fabioscsilva  already removed this font-family right?

Reply:
Yes, this is not need. We have it on body

Reply:
Done!